torchaudio/functional.py

```diff
@@ -858,6 +858,162 @@ def compute_deltas(specgram, win_length=5, mode="replicate"):
     return output
 
 
+def gain(waveform, gain_db=1.0):
+    # type: (Tensor, float) -> Tensor
+    r"""Apply amplification or attenuation to the whole waveform.
+
+    Args:
+       waveform (torch.Tensor): Tensor of audio of dimension (channel, time).
+       gain_db (float) Gain adjustment in decibels (dB) (Default: `1.0`).
+
+    Returns:
+       torch.Tensor: the whole waveform amplified by gain_db.
+    """
+    if (gain_db == 0):
+        return waveform
+
+    ratio = 10 ** (gain_db / 20)
+
+    return waveform * ratio
+
+
+def scale_to_interval(waveform, interval_max=1.0):
```

Comment:
Do we want to expose and maintain this as a separate function? I worry that it's too simple to warrant an explicit abstraction. We could preprend an underscore for now to signal that it is private. I also don't see it used anywhere outside of the tests.

Reply:
Let's remove this function from this PR. We can have a separate PR for it if we want to keep the history.